### ISSUE TITLE
COLMAP Nerfstudio still cannot pick correct model

### ISSUE THREAD

**Issue by abrahamezzeddine** [nerfstudio-project/nerfstudio#3435]
Hello,

Please see output below;

```
[2024/09/18 16:25:16 hloc INFO] Reconstructed 2 model(s).
[2024/09/18 16:25:16 hloc INFO] Largest model is #1 with 306 images.
[2024/09/18 16:25:16 hloc INFO] Reconstruction statistics:
Reconstruction:
        num_reg_images = 306
        num_cameras = 1
        num_points3D = 154700
        num_observations = 586846
        mean_track_length = 3.79345
        mean_observations_per_image = 1917.8
        mean_reprojection_error = 1.38483
        num_input_images = 306
[16:25:16] 🎉 🎉 🎉 All DONE 🎉 🎉 🎉                                                images_to_nerfstudio_dataset.py:135
           Starting with 306 images                                                  images_to_nerfstudio_dataset.py:138
           Colmap matched 3 images                                                   images_to_nerfstudio_dataset.py:138
           COLMAP only found poses for 0.98% of the images. This is low.             images_to_nerfstudio_dataset.py:138
           This can be caused by a variety of reasons, such poor scene coverage,
           blurry images, or large exposure changes.
```
Is there a reason why nerfstudio is unable to pick the right model? Obviously, the COLMAP reconstruction works well. However, folder structure seems very weird.

This folder below contains cameras.bin, database, images.bin and points3D.bin
output\colmap\sparse\0. These files only contains 3 images, with 989 points.

Inside this folder,
output\colmap\sparse\0, there is an additional folder called models, which contains additional folder 0 and 1.
\output\colmap\sparse\0\models\0
\output\colmap\sparse\0\models\1

Folder 0 contains an additional cameras.bin, images.bin and points3D.bin. This model contains 306 images and 154700.
Folder 1 is empty and contains no further files.

Could anyone please look into this bug as to why Nerfstudio is unable to pick the right model? Using this command;
`ns-process-data images --sfm-tool hloc --num-downscales 0 --data projekt/Teknikrum/images --output-dir projekt/Teknikrum/output --feature-type superpoint_max`

Thank you.

**Comment by massyzs** [comment]
Yeah, I'm facing the same problem. Dataset is from some 3D dataset and the images I think are likely to be ok

**Comment by kuaile258890** [comment]
> Yeah, I'm facing the same problem. Dataset is from some 3D dataset and the images I think are likely to be ok

I think it might be a version issue with COLMAP. Downgrading COLMAP from version 3.11 to 3.8 resolved the problem.

**Comment by vectorman1** [comment]
Facing this issue on COLMAP 3.8 as well. 

```
root@2b0bb4154f5b:/# colmap
COLMAP 3.8 -- Structure-from-Motion and Multi-View Stereo
              (Commit 43de802c on 2023-01-31 with CUDA)
```

![image](https://github.com/user-attachments/assets/588f5ad1-d854-41c5-be56-56f677cff6fb)

**Comment by SuzukazeAoba13** [comment]
> Facing this issue on COLMAP 3.8 as well.
> 
> ```
> root@2b0bb4154f5b:/# colmap
> COLMAP 3.8 -- Structure-from-Motion and Multi-View Stereo
>               (Commit 43de802c on 2023-01-31 with CUDA)
> ```
> 
> ![image](https://private-user-images.githubusercontent.com/6563116/398959682-588f5ad1-d854-41c5-be56-56f677cff6fb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4NTIxNDMsIm5iZiI6MTczNjg1MTg0MywicGF0aCI6Ii82NTYzMTE2LzM5ODk1OTY4Mi01ODhmNWFkMS1kODU0LTQxYzUtYmU1Ni01NmY2NzdjZmY2ZmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDExNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMTRUMTA1MDQzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjNjYmZhN2MwYTk4MmJmODQxMTczNmY5YTcxOTY2YzY5ZjllZmVhYjFiMWIxOTQ0YjVkM2Q2MDUxNWFlNjFkZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Go7XlY3ujz6Tc7BKZSfPAO3GMKmaeLQJMOjFxXTegmk)
Hello, has this issue been resolved?
